engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -224,3 +224,9 @@ CALL `cloud`.`IDEMPOTENT_MODIFY_COLUMN_CHAR_SET`('vpc', 'display_text', 'VARCHAR
 CALL `cloud`.`IDEMPOTENT_MODIFY_COLUMN_CHAR_SET`('vpc_offerings', 'name', 'VARCHAR(255)', 'DEFAULT NULL COMMENT \'vpc offering name\'');
 CALL `cloud`.`IDEMPOTENT_MODIFY_COLUMN_CHAR_SET`('vpc_offerings', 'unique_name', 'VARCHAR(64)', 'DEFAULT NULL COMMENT \'unique name of the vpc offering\'');
 CALL `cloud`.`IDEMPOTENT_MODIFY_COLUMN_CHAR_SET`('vpc_offerings', 'display_text', 'VARCHAR(255)', 'DEFAULT NULL COMMENT \'display text\'');
+
+-- Add support for VMware 8.0u2 (8.0.2.x) and 8.0u3 (8.0.3.x)
```

Comment:
@sureshanaparti 
have you checked if there are new supported guest OS in 8.0.2.x and 8.0.3.x ?

Reply:
no new supported guest OSes @weizhouapache